.travis.yml

```diff
@@ -21,6 +21,7 @@ before_install:
   - if [[ $coverage = 1 ]]; then mkdir -p build/logs build/cov; fi
   - if [[ $coverage = 1 ]]; then wget https://phar.phpunit.de/phpcov.phar; fi
   - if [[ $coverage = 1 ]]; then wget https://github.com/satooshi/php-coveralls/releases/download/v1.0.1/coveralls.phar; fi
+  - if [[ $lint = 1 ]]; then wget https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/v1.12.2/php-cs-fixer.phar; fi
```

Comment:
I agree but:

    wget http://get.sensiolabs.org/php-cs-fixer.phar
    php php-cs-fixer.phar -V  
PHP CS Fixer version 1.12.2 by Fabien Potencier and Dariusz Ruminski (baa7112)

It's not the 2.0 branch for now. And if the CI start to fail because of the 2.0 release, it will be time to update :)
Anyway I've nothing against using composer (it will just slow down a bit our builds).